Esignet/scripts/Esignet_Test_Script.jmx

```diff
@@ -165,36 +164,59 @@
             <stringProp name="Argument.value">804296</stringProp>
             <stringProp name="Argument.metadata">=</stringProp>
           </elementProp>
-
+          <elementProp name="countryCodeCambd" elementType="Argument">
+            <stringProp name="Argument.name">countryCodeCambd</stringProp>
+            <stringProp name="Argument.value">+855</stringProp>
+            <stringProp name="Argument.metadata">=</stringProp>
+          </elementProp>
+          <elementProp name="verifyChallenge" elementType="Argument">
+            <stringProp name="Argument.name">verifyChallenge</stringProp>
+            <stringProp name="Argument.value">111111</stringProp>
+            <stringProp name="Argument.metadata">=</stringProp>
+          </elementProp>
+          <elementProp name="consent" elementType="Argument">
+            <stringProp name="Argument.name">consent</stringProp>
+            <stringProp name="Argument.value">AGREE</stringProp>
+            <stringProp name="Argument.metadata">=</stringProp>
+          </elementProp>
+          <elementProp name="language" elementType="Argument">
+            <stringProp name="Argument.name">language</stringProp>
+            <stringProp name="Argument.value">eng</stringProp>
+            <stringProp name="Argument.metadata">=</stringProp>
+          </elementProp>
+          <elementProp name="clientId" elementType="Argument">
```

Comment:
Sign Up Service - Verify Challenge (Execution) - > Sign Up Service - Verify Challenge Endpoint API - request body format looks different comparing the stoplight. Type is missing, challengeInfo in an array.

Reply:
To be taken care in other thread groups also

Reply:
Done